### PR TITLE
UI

### PR DESCRIPTION
**Pull request by SemyonKorostik**

### CODE REVIEW

File: UI/index.html [hunk 1]
```diff
@@ -1,5 +1,3 @@
-<!DOCTYPE html>
-
 <html>
```
Comment (YauheniyaBasaranovich) [inline UI/index.html:1]: Why did you remove content?

Comment (SemyonKorostik) [reply UI/index.html:1]: Stepan Miakchilo wrote me what needs to be corrected in the task. Obviously, I misunderstood him.

Comment (YauheniyaBasaranovich) [reply UI/index.html:1]: It was written in different PR, here please revert, so I'll have possibility to check the task 🙂

Comment (SemyonKorostik) [reply UI/index.html:1]: I'm having problems with the local repository. Is it possible to re-create a repository in github?

Comment (SemyonKorostik) [reply UI/index.html:2]: I solved my problem. I only needed to fix index.html?

Comment (YauheniyaBasaranovich) [reply UI/index.html:2]: Sorry, missed your comment 😢 
Yes, I see everything else is here already

File: UI/index.html [hunk 2]
```diff
@@ -0,0 +1,973 @@
...
+</head>
+
+<body>
+
+<header>
+    <div class="header">
+        <img class="logo" src="lib\pic\logo.png"/>
+        <h1 class="title">АГОРА</h1>
+        <h2 class="subtitle">Здесь можно найти всё</h2>
+        <input type="button" class="add_ad" value="Добавить объявление"/>
```
Comment (YauheniyaBasaranovich) [inline UI/index.html:18]: It's not a big deal here, but in case you don't need to support old browsers (and you actually don't), it's better to use `button` tag.

File: UI/index.html [hunk 3]
```diff
@@ -0,0 +1,973 @@
...
+                    </select>
+                </p>
+
+                <ul class="filter_list_hashtag">
+                    <li>
+                        <aside>
+                            <input type="checkbox" class="checkbox">
+                            хештег
+                        </aside>
+                    </li>
```
Comment (YauheniyaBasaranovich) [inline UI/index.html:72]: Could you please explain how filters are supposed to work in your app?

File: UI/index.html [hunk 4]
```diff
@@ -0,0 +1,973 @@
...
+                    Наушники Magic Earbuds (аквамариновый голубой)124 hjk hjk kНаушники Magic Earbuds (аквамариновый
+                    голубой)
+                </div>
+
+                <div class="short_description">
+                    беспроводные наушники с микрофоном, внутриканальные, портативные, Bluetooth 5.0, быстрая зарядка,
+                    время работы 3.5 ч
+                </div>
+
+                <div class="site_link">
```
Comment (YauheniyaBasaranovich) [inline UI/index.html:199]: It definitely should be `a` tag, not `div`

**Review by YauheniyaBasaranovich — APPROVED** [review]

File: UI/index.html (hunk 3 above)
Comment (SemyonKorostik) [reply UI/index.html:72]: Ads are filtered by date (from to), by hashtags (you can select multiple), and by vendor (you can select only one). I replaced checkbox with radio when selecting a vendor.
